dvc/state.py

```diff
@@ -473,3 +479,25 @@ def remove_unused_links(self, used):
                 self.LINK_STATE_TABLE, ",".join(["?"] * len(chunk_unused))
             )
             self._execute(cmd, tuple(chunk_unused))
+
+
+def connect_sqlite(filename, options):
```

Comment:
Do these methods really need to be public and outside of State class?

Reply:
These functions has nothing to do with class.

Reply:
@Suor maybe then at least make them private with `_` prefix?